src/main/java/com/example/inventoryservice/dto/IngredientDto.java

```diff
@@ -0,0 +1,34 @@
+package com.example.inventoryservice.dto;
+
+import com.example.inventoryservice.entity.Restaurant;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.DecimalMin;
+import javax.validation.constraints.NotBlank;
+import java.math.BigDecimal;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class IngredientDto {
```

Comment:
[medium] When this is used inside `RecipeIngredientDto` and being returned to the user, we might want to exclude some fields to avoid repetition and to get rid of unnecessary data. E.g. when querying a recipe I get full info about its underlying ingredients, like the current amount in the stock. I don't want that, I just need the amount of the ingredient required for a recipe and short info about the ingredient (name, measure unit, and, if needed, id) while the rest is just redundant. So, you might want to review your API to return only the data relevant for a request rather than giving everything you have.